ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/response/FullAdmissionResponseMapperTest.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.cmc.ccd.mapper.response;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import uk.gov.hmcts.cmc.ccd.config.CCDAdapterConfig;
+import uk.gov.hmcts.cmc.ccd.domain.CCDYesNoOption;
+import uk.gov.hmcts.cmc.ccd.domain.response.CCDFullAdmissionResponse;
+import uk.gov.hmcts.cmc.domain.models.response.FullAdmissionResponse;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleParty;
+
+import java.time.LocalDate;
+
+import static uk.gov.hmcts.cmc.ccd.assertion.Assertions.assertThat;
+import static uk.gov.hmcts.cmc.ccd.domain.CCDPaymentOption.FULL_BY_SPECIFIED_DATE;
+import static uk.gov.hmcts.cmc.ccd.util.SampleData.getCCDPartyIndividual;
+import static uk.gov.hmcts.cmc.domain.models.PaymentOption.IMMEDIATELY;
+import static uk.gov.hmcts.cmc.domain.models.response.YesNoOption.NO;
+
+@SpringBootTest
+@ContextConfiguration(classes = CCDAdapterConfig.class)
+@RunWith(SpringJUnit4ClassRunner.class)
+public class FullAdmissionResponseMapperTest {
+
+    @Autowired
+    private FullAdmissionResponseMapper mapper;
+
+    @Test
+    public void shouldMapFullAdmissionResponseToCCD() {
+        //given
+        FullAdmissionResponse fullAdmissionResponse = FullAdmissionResponse.builder()
```

Comment:
Please add tests for remaining fields like statements of truth, statement of means, free mediation, etc.

Reply:
done